diff --git a/CMakeLists.txt b/CMakeLists.txt
index 3d20b80..e854f45 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -178,10 +178,16 @@ add_library(kfr_io ${KFR_IO_SRC})
 target_link_libraries(kfr_io kfr)
 target_compile_definitions(kfr_io PUBLIC KFR_ENABLE_FLAC=1)
 
-install(TARGETS kfr kfr_io ARCHIVE DESTINATION lib)
+install(TARGETS kfr kfr_io
+	RUNTIME DESTINATION bin
+        LIBRARY DESTINATION lib
+        ARCHIVE DESTINATION lib)
 
 if (ENABLE_DFT)
-    install(TARGETS kfr_dft ARCHIVE DESTINATION lib)
+    install(TARGETS kfr_dft
+	RUNTIME DESTINATION bin
+        LIBRARY DESTINATION lib
+        ARCHIVE DESTINATION lib)
 endif ()
 
 install(DIRECTORY include/kfr DESTINATION include)
